ux.symfony.com/src/Service/Changelog/ChangelogProvider.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Service\Changelog;
+
+use Psr\Cache\CacheItemInterface;
+use Symfony\Contracts\Cache\CacheInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+final class ChangelogProvider
+{
+    public function __construct(
+        private readonly HttpClientInterface $httpClient,
```

Comment:
Since the HttpClient is used directly, it should be added to the `composer.json` as a direct dependency

Reply:
I did not even checked, i was convinced we already used it ... :) Fixed